src/bcs/CoupledOutflowBC.C

```diff
@@ -0,0 +1,52 @@
+#include "CoupledOutflowBC.h"
+
+registerMooseObject("MoltresApp", CoupledOutflowBC);
+
+template <>
+InputParameters
+validParams<CoupledOutflowBC>()
+{
+  InputParameters params = validParams<IntegratedBC>();
+  params.addRequiredCoupledVar("uvel", "x direction velocity");
```

Comment:
This is not a common practice in moose. It doesn't define a default value because it is a required variable for the simulation to run. If the variable has not been defined, the simulation won't run. So, defining a default value is redundant.

Reply:
I didn't provide a default value because it would inform the user that he/she/they did not provide a coupled velocity variable. We have the existing OutflowBC.C for constant velocity values.